### PR TITLE
Use `yield from` where possible

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**

### CODE REVIEW

**Review by tacaswell — APPROVED** [review]

File: other/unterm-generator.py [hunk 1]
```diff
@@ -7,8 +7,7 @@
 
 def yield_groups(filename):
     with h5py.File(filename, 'r') as fh:
-        for group in fh:
-            yield group
```
Comment (takluyver) [inline other/unterm-generator.py:11]: Please leave this one alone - it's a script that checked for a very specific bug, and even a small change could affect how it behaves. It's not part of the h5py package, the code style really doesn't matter.

(Alternatively, find a version of h5py from before the bug was fixed, and verify that the modified script still shows the same bug. But that's not really a good use of time. :wink: )

Comment (DimitriPapadopoulos) [reply other/unterm-generator.py:11]: I will follow your adivce. Just give me some time to revert one of the files.

Comment (DimitriPapadopoulos) [reply other/unterm-generator.py]: Done.

**Comment by takluyver** [comment]
Thank you!

**Pull request merged**